packages/web/src/components/NavBar/NavProfileMenu.tsx

```diff
@@ -10,28 +10,45 @@ import {
   Avatar,
 } from '@chakra-ui/react';
 
-export const NavProfileMenu: React.FC = () => (
-  <Menu variant="navbar">
-    <MenuButton as={Button}>
-      <Avatar size={'sm'} />
-    </MenuButton>
-    <MenuList>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/profile">
-          Edit Profile
-        </Link>
-      </MenuItem>
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/app/contributions">
-          View Contributions
-        </Link>
-      </MenuItem>
-      <MenuDivider />
-      <MenuItem>
-        <Link variant="navbarMenuItem" href="/api/auth/github/logout">
-          Log Out
-        </Link>
-      </MenuItem>
-    </MenuList>
-  </Menu>
-);
+export const NavProfileMenu: React.FC = () => {
+  const [currentUserId, setCurrentUserId] = React.useState<string>('');
```

Comment:
When you set the initial value, React will infer the type so you don't need to specify `string` here 
```suggestion
  const [currentUserId, setCurrentUserId] = React.useState('');
```